microprot/scripts/split_search.py

```diff
@@ -550,3 +551,54 @@ def mask_sequence(hhsuite_fp, fullsequence_fp, subsequences_fp=None,
                 for type_ in results}
     except IOError:
         raise IOError('Cannot write to file "%s"' % subsequences_fp)
+
+
+def pretty_output(mask_out):
+    for key in mask_out.keys():
+        print(key)
+        for i in range(len(mask_out[key])):
+            print('\t>', mask_out[key][i][0], '\n\t', mask_out[key][i][1])
```

Comment:
the more pythonic way would be:
print('\t>%s\n\t%s' % (mask_out[key][i][0], mask_out[key][i][1])